department-of-veterans-affairs/vets-website
# [MHV-55186] Update all V1 components to V3 web components on SM MHV

## Summary
- #### V1 Web-Components are retired. This update switches all va-components to utilize the V3 uswds web-components.

> Explanation: 
>The components are currently built so one can flip between v1 and v3. But that bloats the components with code that isn't being used. Once v1 is deprecated we'll commence removing that code.
>
>Also, for the sake of consistency we want to move [va.gov](http://va.gov/) to v3 components within a reasonable time frame. Thus hanging on to v1 versions makes that process take longer. @humancompanion-usds 
>

## Related issue(s)


### [[MHV-55186]](https://jira.devops.va.gov/browse/MHV-55186) Update Retired V1 components to V3


## Testing done

- _Manual Testing_
> To test, start on the 'Messages' landing page and click through all elemental components, (i.g. Accordion, Alerts, Links, etc.), and use google DOM to check if each component contains "uswds" attribute.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Va.gov - Secure Messaging)*

## Acceptance criteria

#### AC1 All components have been upgraded to V3
#### AC2 V3 Components to test:
- Va-Accordion
- Va-Additional-Info
- Va-Alert
- Va-Button
- Va-Modal
- Va-Pagination
- Va-Radio
- Va-Select
- Va-Text-Input
- Va-Textarea

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
